README.md

```diff
@@ -1 +1,58 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
+
+
+## FAQ
+
+#### What are the priorities for the project?
+
+Well... We need a working demo. That is the most important thing for now. As [bopke](https://github.com/bopke) (project leader) [mentioned](https://github.com/FunnyGuilds/FunnySectors/issues/6) we need a working demo to avoid burnout.
+
+#### What technology are you going to start with?
+
+* Project language: [Java](https://www.java.com/en/)
+* Test language: [Kotlin](https://kotlinlang.org/)
+* Message broker: [Redis](https://redis.io/)
+* Minecraft platform: [Spigot](https://www.spigotmc.org/)
+* Proxy: [Velocity](https://velocitypowered.com/)
+
+All the technologies have been discussed [here](https://github.com/FunnyGuilds/FunnySectors/issues)
+
+### Are you planing on adding support for XYZ?
+
+We are planing to have a good abstraction layer so adding anything is not that hard. If you really care you can create issue or pull request. All contributions are welcome!
+
+### Is there any way to contact you?
+
+Yes! You can join project discord [here](https://discord.gg/CYvyq3u)
+
+
+
+
+## Roadmap
+
+- create project (setup [modules](https://github.com/FunnyGuilds/FunnySectors/issues/9))
+
+- create abstraction layer for message broker, database
+
+- create spigot "one source of truth" (Database as a way of storing players equipment)
+
+- split map into small areas (chunks) and add a way to send player between them
+
+- send player equipment using message broker and recive it on the other end
+
+- release first demo version
```

Comment:
I don't think that first step is really worth mention at all
```suggestion
- create abstraction layer for message broker, database

- create spigot "one source of truth" (Database as a way of storing players equipment)

- split map into small areas (chunks) and add a way to send player between them

- send player equipment using message broker and recive it on the other end

- release first demo version
```
Apart from that, it looks good now